apps/web/src/composables/users.ts

```diff
@@ -1,5 +1,5 @@
 import { onMounted, ref } from 'vue'
-import { AddAccountOptions, ProviderString, RemoveAccountOptions, UserWithAccounts, Account } from '@casimir/types'
+import { AddAccountOptions, ProviderString, RemoveAccountOptions, UserWithAccounts, Account, ExistingUserCheck } from '@casimir/types'
```

Comment:
Check me on the `ExistingUserCheck` type I added. Not sure if it's best practice or if there is a better design pattern for this (used in an api call in this file within `checkIfPrimaryUserExists` method).